### ISSUE TITLE
Number field accessibility

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#7367]
### Description

- Prefix and suffix should be linked to input using visually-hidden instructions text and aria-describedby (similar to lightswitch field).
- Min value, max value, and decimal points are not clear (See #7369)

### Steps to reproduce

1. Test a number field.

### Additional info

- Craft version: 3.5.17.1
